docs/advanced/tracking.story.mdx

```diff
@@ -0,0 +1,67 @@
+import { Meta, Intro, Story } from '../../.storybook/components';
+
+<Meta title="Advanced/Build-in Tracking" />
+
+# Build-in Tracking
+
+<Intro>
+  Working along with{' '}
+  <a href="https://github.com/sumup-oss/collector">Collector</a>, button-like
+  components and meanful UI operation in Circuit UI will come along with
+  build-in tracking method. Components like Button, Tag and Hambuger will invoke
+  a pre-set dispatch function to trigger tracking event. For more detail
+  concept, please refer to our collector project.
```

Comment:
Documentation should be written in present tense, not future tense, since it describes the state-is not as it will be.